homeassistant/components/snoo/strings.json

```diff
@@ -0,0 +1,59 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "username": "Your Happiest Baby username or email",
+          "password": "Your Happiest Baby password"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "last_event": {
+        "name": "Last Event",
+        "state": {
+          "activity_button": "Activity button",
+          "timer": "Timer",
+          "power_button": "Power button",
+          "cry": "Cry",
+          "command": "Command",
+          "safety_clip": "Safety clip",
+          "long_activity_press": "Long activity press",
+          "restart": "restart",
+          "initial_status_requested": "initial_status_requested",
+          "status_requested": "status_requested"
+        }
+      },
```

Comment:
So wait, this is push based right? Why don't we do for an event entity?

Reply:
I'm open to it, but i'm not sure if this fully applies:

"The event entity captures these events in the physical world and makes them available in Home Assistant as an entity."

Some of these events are not physical world based, idk if that's a problem. I've never actually done a event entity before. But this is just the data that comes along with the message on the pubnub channel, no data right now is separated and all comes in the same and the Coordinator just handles all of it.

Reply:
It sounds like a big mashup of different events, but can you link it to the device? so for example maybe it makes sense to have 1 event entity for the activity button and register the events `activity_button` and `long_activity_press` to that event with types "press" and "long_press". Maybe this way it can resemble the device in a better way

Reply:
So should this just be ripped out? Not have a sensor at all and instead take the data and put it where it makes sense?

i.e.

An event for power

An event for activity button

A binary sensor for is crying?

etc?

Or those things in addition to a regular sensor showing all of them?

Reply:
but crying is not a state, rather an event. Unless we get an event for start of crying and event for stop of crying

Reply:
So regardless - just nuke this for now? No sensor and replace with at the very least one event entity?

Reply:
I think that's the best yes